src/utilities/helpers/focus-on-first-invalid-choice/index.ts

```diff
@@ -0,0 +1,19 @@
+import {
+  nextTick,
+  type Ref,
+} from 'vue';
+import UiMultipleChoices from '../../../components/organisms/UiMultipleChoices/UiMultipleChoices.vue';
+
+export default function focusOnInvalidChoice(invalidChoices: Ref<InstanceType<typeof UiMultipleChoices> | null>) {
+  const focusInvalidInput = async () => {
+    if (!invalidChoices.value) return;
+
+    const { focusInvalidChoice } = invalidChoices.value;
+
+    await nextTick();
```

Comment:
Is nextTick really necessary or is it just leftover from `.clear()` solution?

Reply:
Leftover. Removed